android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -125,17 +126,21 @@ public String getCarrier() {
 
     PackageManager packageManager = this.reactContext.getPackageManager();
     String packageName = this.reactContext.getPackageName();
-
+    String applicationName = this.reactContext.getApplicationInfo().loadLabel(this.reactContext.getPackageManager()).toString();
+    
     constants.put("appVersion", "not available");
+    constants.put("appName", "not available");
     constants.put("buildVersion", "not available");
     constants.put("buildNumber", 0);
 
     try {
+      PackageInfo packageInfo = packageManager.getPackageInfo(packageName, 0);
       PackageInfo info = packageManager.getPackageInfo(packageName, 0);
       constants.put("appVersion", info.versionName);
       constants.put("buildNumber", info.versionCode);
       constants.put("firstInstallTime", info.firstInstallTime);
       constants.put("lastUpdateTime", info.lastUpdateTime);
+      constants.put("appName", packageManager.getApplicationLabel(applicationInfo).toString());
```

Comment:
I am getting undefined applicationInfo, this is probably just `info`

Reply:
Hmm, probably happened in a commit which was resolving the conflicts. I was not careful enough. Can anyone make quick fix or do I need to create another pull request?

Reply:
yeah https://github.com/rebeccahughes/react-native-device-info/pull/295